evap/staff/views.py

```diff
@@ -466,15 +466,20 @@ def semester_export(request, semester_id):
     if formset.is_valid():
         include_not_enough_voters = request.POST.get('include_not_enough_voters') == 'on'
         include_unpublished = request.POST.get('include_unpublished') == 'on'
+        degrees_list = []
         course_types_list = []
         for form in formset:
+            if 'selected_degrees' in form.cleaned_data:
+                degrees_list.append(form.cleaned_data['selected_degrees'])
             if 'selected_course_types' in form.cleaned_data:
                 course_types_list.append(form.cleaned_data['selected_course_types'])
```

Comment:
when do those ifs return false? can one of the ifs return false and the other true?

Reply:
they would probably return false if nothing was selected but the form is not valid then so this part of the code wouldn't be reached. so it can be removed.